release-content/0.14/release-notes/10690_Upgrade_to_Taffy_04.md

```diff
@@ -1 +1,5 @@
-TODO
+[`taffy`](https://crates.io/crates/taffy), the UI layout crate that we rely on, has had a major update.
```

Comment:
Doesn't affect content in this PR (except the filename), but it's worth noting that the upgrade is now to 0.5 rather than 0.4